Source/pages/js/utils.js

```diff
@@ -1,51 +0,0 @@
-/**
```

Comment:
Yes. This file was part of the first refactor done for options page and however with the later changes made this file is no longer in use.